src/nvim/api/command.c

```diff
@@ -149,8 +149,6 @@ Dictionary nvim_parse_cmd(String str, Dictionary opts, Error *err)
     }
     ADD(range, INTEGER_OBJ(ea.line2));
     PUT(result, "range", ARRAY_OBJ(range));
-  } else {
-    PUT(result, "range", ARRAY_OBJ(ARRAY_DICT_INIT));
```

Comment:
Another approach is to use an empty array if range is not specified, but omit if cannot cannot accept a range. Will this be better (e.g. more consistent with when `count` is present and allows telling whether command supports a range)?

Reply:
I think that would be better, yes